### PR TITLE
React1 week1

### PR DESCRIPTION
**Pull request by DennisAgyekum**
react1-week1

### CODE REVIEW

**Comment by DennisAgyekum** [comment]
Screenshot 2024-10-10 at 06 26 51
Screenshot 2024-10-10 at 06 27 30
Screenshot 2024-10-10 at 09 00 39

File: app/about_us/OurCrew.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import styles from "./page.module.css";
+
+const crewFields = [
```
Comment (Hadis-jamali) [inline app/about_us/OurCrew.js]: for name crewMembers is better

File: app/about_us/OurPartners.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+export default function OurPartners() {
+    return (
+      <>
+        <h3 className={styles.heading}>We collaborate with some of the most respected names in the space and technology industries to make every journey extraordinary.</h3>
+        <div className={styles.logoContainer}>
+          {
+          partners.map((partner) => (
+            <div className={styles.logoDiv} key={partner.companyName}>
+              <img src={partner.logoUrl} alt={`logo of ${partner.companyName}`} className={styles.logoImage} />
```
Comment (Hadis-jamali) [inline app/about_us/OurPartners.js:46]: Nice job on dynamically setting the alt text for the logo!

File: app/about_us/page.module.css [hunk 3]
```diff
@@ -3,3 +3,123 @@
...
+
+.value {
+  border-bottom: 1px solid rgba(88, 95, 137, 0.755);
+}
+
+.valueTitle {
+  color: gray;
+  font-size: 120px;
+  font-weight: 700;
+  padding: 20px 30px 20px 30px;
```
Comment (Hadis-jamali) [inline app/about_us/page.module.css]: When you provide only two values for padding, the first value (20px) applies to the top and bottom, and the second value (30px) applies to the left and right.

File: app/about_us/page.module.css [hunk 4]
```diff
@@ -3,3 +3,123 @@
...
+}
+
+.ourValues {
+  font-weight: 600;
+}
+
+.valueDescription {
+  line-height: 22px;
```
Comment (Hadis-jamali) [inline app/about_us/page.module.css]: try using rem instead of px for padding, margins, and font sizes....

**Review by Hadis-jamali — COMMENTED** [review]
good job Dennis,  Keep up the awesome work

File: app/about_us/OurCrew.js (hunk 1 above)
Comment (DennisAgyekum) [reply app/about_us/OurCrew.js]: ok will change that, thanks

File: app/about_us/page.module.css (hunk 3 above)
Comment (DennisAgyekum) [reply app/about_us/page.module.css]: I am going to make the changes.

File: app/about_us/OurPartners.js (hunk 2 above)
Comment (DennisAgyekum) [reply app/about_us/OurPartners.js:46]: Thank you.

**Pull request merged**